tests/e2e/commands/fillInput.js

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+exports.command = function fillInput(element, string) {
+    return this
+        .waitForElementVisible(element, 3000)
```

Comment:
These timeouts seem really long. I'm concerned that they could cause the test suite to take a really long time to run.

Reply:
Yeah, actually I have longer time of `DEFAULT_WAIT: 20000`. That time period is maximum (which we can change later once app loading performance improved).
Loading of apps in the actual test browser is not consistent (sometimes it takes more than 10 sec. based on my observation) and having shorter time delay might cause flaky test.  
The way it works is that once the element is visible, it will do the assertion as soon as possible. If it reached the wait time, then it will fail and will do the retry once (`--suiteRetries 1`).